v22.2/online-schema-changes.md

```diff
@@ -64,17 +64,23 @@ If a schema change fails, the schema change job will be cleaned up automatically
 
 The declarative schema changer is the next iteration of how schema changes will be performed in CockroachDB. By planning schema change operations in a more principled manner, the declarative schema changer will ultimately make transactional schema changes more stable. You can identify jobs that are using the declarative schema changer by running [`SHOW JOBS`](show-jobs.html) and finding jobs with a `job_type` of `NEW SCHEMA CHANGE`.
 
-The following statements use the declarative schema changer by default:
+The following statements can use the declarative schema changer, and are enabled to do so by default:
```

Comment:
I didn't know about this section. This is good. I think we need to say more about when it is used. In particular, the declarative schema changer is not used in an explicit transaction or a multi-statement implicit transaction; it is only used by default for single statement, implicit transactions. 

There's also a list of special cases for some of these statements where it is not used. We should document them. Let me make a list now.